utils/mempool.js

```diff
@@ -0,0 +1,41 @@
+const axios = require('axios')
+const MEMPOOL_URL = process.env.MEMPOOL_URL || 'https://mempool.space'
+const MEMPOOL_RETRY_URL = process.env.MEMPOOL_RETRY_URL || 'https://mempool.deezy.io'
```

Comment:
```suggestion
const MEMPOOL_RETRY_URL = process.env.MEMPOOL_RETRY_URL || 'https://mempool-bk.deezy.io'
```
I think this is the proper one (cc @habibitcoin )?